samples/deployment/test-network/compose.yaml

```diff
@@ -41,6 +43,8 @@ services:
       - CHAINCODE_PKG_ID=${ORG2_ECC_PKG_ID}
       - FABRIC_LOGGING_SPEC=${FABRIC_LOGGING_SPEC:-DEBUG}
       - SGX_MODE=${SGX_MODE:-SIM}
+      - RUN_CCAAS=true
+      - FPC_ENABLED=true
     networks:
```

Comment:
those flags are specific of the cc-tools demo. Can we set these params maybe via you commands scripts instead of setting them here as default?

Reply:
I was aiming to remove this script anyway as the tutorial should have all the script's commands. Instead of adding the commands to the PR I can remove it and require it in the tutorial that you need to add those to the env vars